### ISSUE TITLE
Subject: Genre and spacing

### ISSUE THREAD

**Issue by joycebcat** [pulibrary/orangelight#251]
If there is an extra space between the subfield code and the content of the subfield, it appears in the Genre list as a separate listing.  Can we set up the system to strip whitespaces before creating the list?  This may be occurring in other facet lists as well, I haven't checked yet.

**Comment by tampakis** [comment]
In general - I think that there is some cleaning of the data that should happen before adding to the genre facet. Since this was the first full index with the new facet, I didn't know what to expect.

For instance:
http://pulsearch-dev.princeton.edu/catalog/facet/genre_facet?facet.sort=index&q=&search_field=all_fields&utf8=%E2%9C%93

Subject topic doesn't seem too bad, but should also strip out whitespace:
http://pulsearch-dev.princeton.edu/catalog/facet/subject_topic_facet?facet.sort=index&q=&search_field=all_fields&utf8=%E2%9C%93

**Comment by joycebcat** [comment]
I agree. I have been doing some data cleanup in these two areas (some of them are miscoded or invalid), but I won't be able to do a lot before your presentation to LISSC+.  It won't show unless the facet is expanded, and then sorted alphabetically.

**Comment by tampakis** [comment]
Thanks for looking into cleaning up the data @joycebcat. There is some code that I should write too, to handle dirty data a bit better, but that won't be until after the presentation.
